### ISSUE TITLE
Autocomplete for fragments appends entry instead of replacing 

### ISSUE THREAD

**Issue by youngkidwarrior** [zth/vscode-rescript-relay#40]
Small thing, but I feel like it isn't the intended function

When autocompleting fragment names, the existing input is appended, instead of replaced

https://github.com/zth/vscode-rescript-relay/assets/36084652/dc0dd2b8-87d7-4550-8bdc-cf17dadea680

**Comment by zth** [comment]
Agreed! However, this is from the Relay LSP, so it's probably better if you report the issue there.
